packages/engine.test/.npmignore

```diff
@@ -0,0 +1,3 @@
+src
```

Comment:
How is this library supposed to be used? Compiled or without compile?

Reply:
i have no idea

Reply:
what would be the best option? i see it as a simple package